server/routes/search.js

```diff
@@ -14,7 +14,10 @@ router.all(['/search', '/api/search', '/api/form/search'], getForm(form, { submi
 router.get('/api/form/search', (req, res) => res.json(req.form));
 
 router.post(['/search/results', '/api/search/results'],
-    processRequestBody(form().getFields()),
+    async (req, res, next) => {
```

Comment:
this could be abstracted into a middleware component

Reply:
resolved with searchHandler component